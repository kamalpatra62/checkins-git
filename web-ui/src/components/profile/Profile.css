.flex-row {
  display: flex;
  flex-direction: row;
}
.image-div {
  display: flex;
  flex-direction: row;
  margin-left: 50px;
  margin-right: 50px;
}
img {
  border-radius: 50%;
  max-width: 256px;
  max-height: 256px;
}