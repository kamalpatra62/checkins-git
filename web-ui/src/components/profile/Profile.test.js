import React from "react";
import Profile from "./Profile";
import renderer from "react-test-renderer";

it("renders correctly", () => {
  snapshot(<Profile />);
});

it("renders image_url", () => {
  snapshot(<Profile image_url="http://someurl.com/das.png" />);
});
