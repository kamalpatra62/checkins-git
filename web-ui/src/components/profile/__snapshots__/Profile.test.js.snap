// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="flex-row"
>
  <div
    className="image-div"
  >
    <img
      alt="Profile"
      src="https://i.imgur.com/TkSNOpF.jpg"
    />
  </div>
  <div
    className="flex-row"
  >
    <div
      style={
        Object {
          "marginTop": "50%",
          "textAlign": "left",
        }
      }
    >
      <h2
        style={
          Object {
            "margin": 0,
          }
        }
      >
        Name
      </h2>
      <div>
        <p>
          Role
        </p>
        <p>
          Email
        </p>
        <p>
          Current PDL
        </p>
      </div>
    </div>
  </div>
</div>
`;

exports[`renders image_url 1`] = `
<div
  className="flex-row"
>
  <div
    className="image-div"
  >
    <img
      alt="Profile"
      src="http://someurl.com/das.png"
    />
  </div>
  <div
    className="flex-row"
  >
    <div
      style={
        Object {
          "marginTop": "50%",
          "textAlign": "left",
        }
      }
    >
      <h2
        style={
          Object {
            "margin": 0,
          }
        }
      >
        Name
      </h2>
      <div>
        <p>
          Role
        </p>
        <p>
          Email
        </p>
        <p>
          Current PDL
        </p>
      </div>
    </div>
  </div>
</div>
`;
